<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="colorSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>359</number>
    </property>
    <property name="value">
     <number>35</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="colorWidget" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="colorSpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>53</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>359</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QLabel" name="imageFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Color to track (H)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="moveCursorCheckBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Cursor</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="filterSpinBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>53</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="speedSpinBox_X">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>53</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="filterLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>53</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter movement</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpeed_X">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cursor speed X</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dwellClickCheckBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Dwell click</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="minS_SpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>53</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="minV_SpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>53</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>72</number>
    </property>
   </widget>
   <widget class="QLabel" name="minS_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Minimum Saturation (S)</string>
    </property>
   </widget>
   <widget class="QLabel" name="minS_label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Minimum Value (V)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpeed_Y">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cursor speed Y</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="speedSpinBox_Y">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>53</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
   <widget class="QComboBox" name="algorithmComboBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ArUco</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Color</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="algorithmLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithm</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelArUco">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilter_X">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter X</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilter_Y">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter Y</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="filterSpinBox_X">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>53</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="filterSpinBox_Y">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>110</y>
      <width>53</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>colorSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>colorSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>colorSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
